### DISCUSSION TITLE
All weekly assignments for control

### DISCUSSION THREAD

**Discussion by joaochris** [sws2apps/organized-app#125] [Ideas]
Hello. When filling out the assignments, in a congregation with many brothers who are part of it, there is a big issue that is the repetition of the same brother for a certain part. But I see that this is also related to the weekend meeting, as we can assign the same brother two consecutive weeks and he still has part on the weekend. I think there's a lot left.
Mid-week assignments could be linked to weekend assignments to prevent this from happening.

I saw that you opened a feat of public speeches, but do you also intend to post the full weekend meeting?

About the PDF model, we could see which one looks better, I think I have a few.

**Comment by rhahao** [comment]
Hi @joaochris, thanks for inquiry and for sharing the idea. Yes, I plan to have the weekend meeting and public talk scheduler features to be published at the same time. (and that’s after the memorial of course 😁)

And for the template, would like to share it here as well please? The more idea we have, the best we can offer to the user.

**Reply by joaochris** [reply]
![image](https://user-images.githubusercontent.com/76891864/228255510-5a683b71-d53f-48aa-a738-d227b6f4f250.png)
![image](https://user-images.githubusercontent.com/76891864/228256164-7e5b3988-75b5-40d5-8e97-e5a6590d79e0.png)

**Reply by rhahao** [reply]
Hello @joaochris: just to let you know that I do not forget this weekend meeting template. 😉 I will start creating additional templates for the PDF export and let the users choose what is best for them. Thanks.

**Reply by rhahao** [reply]
Hi @joaochris, I’m just doing some cleaning and found this. I believe this should be at least partially solved with the latest CPE update. I hope it will be even better in the new Organized app. 😊

**Reply by fschein** [reply]
If I may show you the model we use here...
![2024-11-04_10-50](https://github.com/user-attachments/assets/f0c7a97c-6ec9-4254-aef4-bc2e0054a8cc)

**Reply by fschein** [reply]
And the weekend meeting
![image](https://github.com/user-attachments/assets/3b80ecff-6cd6-41b6-8bd4-8c9d59e2b977)

**Reply by ux-git** [reply]
Thanks @fschein for sharing! Taking the notes ✍️
